npm/git-cliff/src/index.ts

```diff
@@ -47,8 +50,14 @@ export async function runGitCliff(options: Options, execaOptions?: ExecaOptions)
  * await runGitCliff(["--tag", "1.0.0", "--config", "github", "--topo-order"]);
  * ```
  */
-export async function runGitCliff(args: string[], execaOptions?: ExecaOptions): Promise<ExecaReturnValue<string>>;
-export async function runGitCliff(argsOrOptions: Options | string[], execaOptions?: ExecaOptions): Promise<ExecaReturnValue<string>> {
+export async function runGitCliff(
+  args: string[],
+  execaOptions?: ExecaOptions
+): Promise<ResultPromise<ExecaOptions>>;
+export async function runGitCliff(
+  argsOrOptions: Options | string[],
+  execaOptions?: ExecaOptions
+): Promise<ResultPromise<ExecaOptions>> {
```

Comment:
Is this a breaking change for the users of the Typescript API?

I wonder if there is any changes needed in <https://git-cliff.org/docs/installation/npm>

Reply:
Fair enough! Thanks for checking